server/jetstream_consumer_test.go

```diff
@@ -6517,8 +6525,18 @@ func TestJetStreamConsumerPendingLowerThanStreamFirstSeq(t *testing.T) {
 		if ci.NumAckPending != 0 {
 			return fmt.Errorf("NumAckPending should be 0, got %v", ci.NumAckPending)
 		}
-		if ci.Delivered.Stream != 999_999 {
```

Comment:
This message has never been delivered, so reporting it has been delivered is incorrect. This should report the last sequence of a message that was delivered. However, `o.sseq` should be skipped ahead, which is expected after the `Compact` this test does.